tests/ethpm/backends/test_ipfs_backends.py

```diff
@@ -27,6 +32,15 @@ def test_ipfs_and_infura_gateway_backends_fetch_uri_contents(
         assert contents_dict["package_name"] == "safe-math-lib"
 
 
+def test_local_ipfs_backend():
+    uri = "ipfs://Qme4otpS88NV8yQi8TfTP89EsQC5bko3F5N1yhRoi6cwGV"
+    backend = LocalIPFSBackend("localhost", "8080")
+    with requests_mock.Mocker() as m:
```

Comment:
I might be mistaken, but this seems like an appropriate place to use the mocker to me, since it's testing the `fetch_uri_contents` method which makes an req unique to the `LocalIPFSBackend` class. Using the `DummyIPFSBackend` here makes less sense to me

Reply:
We've already discussed this so I'm just leaving this comment here for the record.  It seems better to change the `LocalIPFSBackend` class to accept an already instantiated `Client` instance from the `ipfsapi` library as the only argument to `__init__` which would get stored in `self.client`.  Then, the `fetch_uri_contents` method would just do  `self.client.cat`.  That would make it easier to inject fake dependencies for that backend for the purpose of testing.  This approach is advocated in this article written by the principal engineer at SendGrid: https://sendgrid.com/blog/when-writing-unit-tests-dont-use-mocks/ .